builder/tart/builder.go

```diff
@@ -69,6 +70,22 @@ func (b *Builder) Prepare(raws ...interface{}) (generatedVars []string, warnings
 		return nil, nil, err
 	}
 
+	fromArgs := []bool {
+		b.config.FromIPSW != "",
+		len(b.config.FromISO) > 0,
+		b.config.VMBaseName != "",
+	}
+
+	fromArgsSet := 0
+    for _, v := range fromArgs {
```

Comment:
Please [use tabs instead of spaces](https://go.dev/doc/effective_go#formatting).

Reply:
Ah, sorry, will fix

Reply:
Done